R/utility.R

```diff
@@ -179,3 +179,28 @@ getJobResult <- function(jobId = "", ...){
 
   return(results)
 }
+
+validateClusterConfig <- function(clusterFilePath){
+  if (file.exists(clusterFilePath)) {
+    pool <- rjson::fromJSON(file = clusterFilePath)
+  }
+  else{
+    pool <- rjson::fromJSON(file = file.path(getwd(), clusterFilePath))
+  }
+
+  stopifnot(!is.null(pool$poolSize))
```

Comment:
There's no stopif unfortunately.. The suggestion they have is if(is.null(pool$poolSize)) stop("poolSize is null")